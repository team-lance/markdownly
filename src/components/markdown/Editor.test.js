import React from 'react';
import { shallow } from 'enzyme';
import Editor from './Editor';

describe('Editor', () => {
  it('matches a snapshot', () => {
    const markdown = '## Some markdown text';
    const updateMarkdown = jest.fn();
    const wrapper = shallow(
      <Editor markdown={markdown}
        updateMarkdown={updateMarkdown}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
