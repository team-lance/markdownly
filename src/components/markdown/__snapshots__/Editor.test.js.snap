// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor matches a snapshot 1`] = `
<textarea
  className="Editor"
  onChange={[MockFunction]}
  value="## Some markdown text"
/>
`;
